doc/api/buffer.md

```diff
@@ -203,7 +203,8 @@ The character encodings currently supported by Node.js include:
 
 * `'binary'` - Alias for `'latin1'`.
 
-* `'hex'` - Encode each byte as two hexadecimal characters.
+* `'hex'` - Encode each byte as two hexadecimal characters. If the number of
+hexadecimal characters is odd, the first incomplete hexadecimal pair is ignored.
```

Comment:
```suggestion
hexadecimal characters is odd, the last character is ignored.
```

Reply:
I think this might be better moving it down into it's own one line paragraph with a simple example

Reply:
If we move it to its own paragraph, we can also add information (if it's not already there) about what happens if the string contains characters that are not hexadecimal (e.g., `'1a7g'`). That can also be in a separate PR, but wouldn't be out of place in this one!

Reply:
@Trott "hexadecimal characters is odd, the last character is ignored." adding this line is enough ?